### PR TITLE
adding content to csv-use-case

### PR DESCRIPTION
**Pull request by MathewBiddle**
A start to the csv use case.

### CODE REVIEW

File: _docs/csv-use-case.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+            <att name="standard_name">time</att>
+            <att name="time_precision">1970-01-01T00:00:00Z</att>
+            <att name="units">yyyy-MM-dd&#39;T&#39;HH:mm:ss&#39;Z&#39;</att>
+        </addAttributes>
+    </dataVariable>
+```
+
+
+* Append xml to the end of datasets.xml
+  * See /python_tools/script1insert.py
```
Comment (fcarvalhopacheco) [inline _docs/csv-use-case.md:397]: Do you have the hyperlink for this?  `/python_tools/script1insert.py`

**Review by fcarvalhopacheco — COMMENTED** [review]
Just some comments that might be useful...

File: _docs/csv-use-case.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+  * Master datasets file is at /usr/local/erddap-gold-standard/erddap/content/datasets.xml
+  * Be sure to put it above the closing </erddapDatasets> tag. Don’t worry, the script takes care of that.
+* Flag the dataset for ERDDAP to load
+  * Eg.$ touch erddap/data/flag/2019_asset_inventory
+* Check ERDDAP for the new dataset (might take a little if it’s a large dataset)
+
+```
+/usr/local/erddap-gold-standard/python_tools$ python
```
Comment (fcarvalhopacheco) [inline _docs/csv-use-case.md:405]: my `~/erddap-gold-standard` repository cloned from ioos doesn't have `~/erddap-gold-standard/python_tools`...

**Review by fcarvalhopacheco — COMMENTED** [review]
Just a question about  `erddap-gold-standard/python_tools`

File: _docs/csv-use-case.md (hunk 1 above)
Comment (MathewBiddle) [reply _docs/csv-use-case.md:397]: Good catch.

**Review by fcarvalhopacheco — COMMENTED** [review]
Check `datasetID` on line 215 and 415

File: _docs/csv-use-case.md (hunk 2 above)
Comment (MathewBiddle) [reply _docs/csv-use-case.md:471]: yeah, that's something special I built. I'll remove those pieces for now.

**Pull request merged**